docs/using-cinder-csi-plugin.md

```diff
@@ -423,6 +423,8 @@ Volumes:
 
 ```
 
+> NOTE: `VolumeAttributes` is optional, if it's not provided, it will be default to `1Gi`
```

Comment:
will default to (delete be)

Reply:
Can we move this after `Create a pod with inline volume`

Reply:
updated both , thanks